README.md

```diff
@@ -12,342 +14,117 @@
 
-- en.json or en-US.json
-- ar.json or ar-DZ.json
-- langs.csv
+```yaml
+assets/translations.cvs
+```
 
-must declare the subtree in your **_pubspec.yaml_** as assets:
+Describe it in your `pubspec.yaml`:
 
 ```yaml
 flutter:
   assets:
-    - {`path`/{languageCode}-{countryCode}.{formatFile}}
+    - assets/translations.csv
 ```
 
-#### EasyLocalization attributes
-
- Attribute | Type  | Default | Required | Description |
-|-----------|-------|---------|-------------|----------|
-| `child` | `Widget` | `null` | `true` | Description |
-| `path` | `String` | `null`  | `true` | Description |
-| `supportedLocales` | `List<Locale>` | `null`  | `true` | Description |
-| `fallbackLocale` | `Locale` | `null` | `false` | Description |
-| `startLocale` | `Locale` | `null`  | `false` | Description |
-| `useOnlyLangCode` | `bool` | `false` | `false` | Description |
-| `assetLoader` | `AssetLoader` | `const RootBundleAssetLoader()`  | `false` | Description |
-| `saveLocale` | `bool` | `true`  | `false` | Description |
-| `preloaderColor` | `Color` | `Colors.white`  | `false` | Description |
-| `preloaderWidget` | `Widget` | `const EmptyPreloaderWidget()`  | `false` | Description |
-
-#### EasyLocalization properties
-
- property | Type | Description |
-|-----------|---------|----------|
-| `tr()` | `String` | Description |
-| `plural()` | `String` | Description |
-| `EasyLocalization.of(context).locale` or `context.locale` | `locale` | set or get locale |
-| `EasyLocalization.of(context).deleteSaveLocale()` or `context.deleteSaveLocale()` | `void` | delete saved locale |
-| `EasyLocalization.of(context).supportedLocales` or `context.supportedLocales` | `locale` | get value of `supportedLocales` |
-| `EasyLocalization.of(context).fallbackLocale` or `context.fallbackLocale` | `locale` | get value of `fallbackLocale` |
-
-#### :fire: `tr()` attributes
-
- Attribute | Type  | Default | Required | Description |
-|-----------|-------|---------|-------------|----------|
-| `context` | `BuildContext` | `null` | `false` | Description |
-| `args` | `List<String>` | `null`  | `false` | Description |
-| `namedArgs` | `Map<String, String>` | `null`  | `false` | Description |
-| `gender` | `String` | `null` | `false` | Description |
-
-example :
+Then in your code use `tr` to translate code:
 
 ```dart
-  Text('title'.tr()),
-  Text('title').tr(),
-  Text(tr('title')),
+  Text(tr('hi'))
 ```
 
-#### :fire: `plural()` attributes
-
- Attribute | Type  | Default | Required | Description |
-|-----------|-------|---------|-------------|----------|
-| `context` | `BuildContext` | `null` | `false` | Description |
-| `value` | `dynamic` | `null`  | `false` | Description |
-| `format` | `NumberFormat` | `null`  | `false` | Description |
-
-example :
+#### A more complete code example
 
 ```dart
-  Text('counter'.plural(counter)),
-  Text('counter').plural(counter),
-  Text(plural('counter',counter)),
-```
-
-#### Loading translations from other resources
```

Comment:
why removed this ??!!

Reply:
i˙ve added a link to easy_localization_loader now